app/models/project.rb

```diff
@@ -1,4 +1,15 @@
 class Project <ApplicationRecord
   validates_presence_of :name, :material
   belongs_to :challenge
+
+  has_many :contestant_projects
+  has_many :contestants, through: :contestant_projects
+
+  def contestant_count
+    contestants.count
+  end
+
+  def average_years_of_experience
+    contestants.average(:years_of_experience).to_f.round(2)
+  end
```

Comment:
This looks good. You may consider leaving the round off of the second method. Maybe this method will serve as a helper method to another calculation you need done in the future... if you want that future calculation to be accurate, you may wish that this method wasn't rounding the average's return value--- just something to think about 🤔

Reply:
I might put the round in the view instead since it's essentially presentation logic.